packages/devtools_app/lib/src/code_size/code_size_controller.dart

```diff
@@ -87,10 +88,28 @@ class CodeSizeController {
     loadFakeDiffTree(
       _oldDiffSnapshotFile.value,
       _newDiffSnapshotFile.value,
-      diffTreeType: newDiffTreeType,
     );
   }
 
+  /// The active tree type used to build the diff treemap.
+  ValueListenable<Key> get activeViewKey {
+    return _activeViewKey;
+  }
+
+  final _activeViewKey = ValueNotifier<Key>(CodeSizeBodyState.snapshotTabKey);
```

Comment:
we shouldn't need this to be a value notifier. Just a boolean that is tracked on the controller. UI code shouldn't need to listen to this